lib/sarif/base_sarif.rb

```diff
@@ -0,0 +1,68 @@
+module Sarif
+  class BaseSarif
+    DEFAULT_URI = "https://github.com/coinbase/salus".freeze
+
+    attr_reader :name, :tool, :results, :scan_report
+
+    def initialize(scan_report)
+      @name = scan_report.scanner_name
+      @scan_report = scan_report
+      @tool = tool_info
+      @results = results_info
+    end
+
+    # Retrieve tool section for sarif report
+    def tool_info
+      {
+        "driver": {
+          "name" => scan_report.scanner_name,
+          "version" => scan_report.version || ""
```

Comment:
You don't need to `|| ""` because the scanner super class sets `version` to empty string.